wunderlist-create-task.js

```diff
@@ -9,19 +9,49 @@ module.exports = function(RED) {
     this.on('input', function(msg) {
       var wunderlistAPI = wunderlistSDK.getApi(node.config, msg);
       var tasks = wunderlistAPI.http.tasks;
+      var reminders = wunderlistAPI.http.reminders;
 
-      tasks.create({
+      var params = {
         'list_id': Number(n.listId || msg.listId),
         'title': n.taskTitle || msg.payload
-      })
+      };
+      // Add due date if in msg
+      if (msg.due_date){
```

Comment:
The convention for node-red is to use `camelCase` so I would change this to `msg.dueDate`.
Of course the Wunderlist still uses `snake_case` so it would mean doing

```javascript
if (msg.dueDate) {
  params.due_date = msg.dueDate;
}
```

Reply:
fixed